documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -1,27 +1,28 @@
 # Creating an SDK Plugin
 
 This document outlines how to use the Performance Toolkit SDK (SDK) to create
-an SDK plugin. A plugin can be used for processing trace files to be used by
-automation, trace extractors, or viewers such as Windows Performance Analyzer.
+an SDK plugin. Plugins can be used to expose data from `DataSource`s for
+automation, trace extractors, or viewers (e.g. Windows Performance Analyzer).
```

Comment:
nit: I think "such as" flows better here since it's at the end of a list. No strong preference, though.